### PR TITLE
[ET-909] Update Move_Tickets.php

### PR DESCRIPTION
**Pull request by prowp**
[ET-909]

This allows the title in the move attendees screen to be filtered to add additional information. An alternate possibility is to add an events tickets specific filter rather than a generalized WP filter.

[ET-909]: https://moderntribe.atlassian.net/browse/ET-909

### CODE REVIEW

**Comment by bordoni** [comment]
Thanks @prowp, our team will take a look into the changes add a changelog and find a release to ship this modification with in the next couple of days.

File: src/Tribe/Admin/Move_Tickets.php [hunk 1]
```diff
@@ -359,7 +359,7 @@ protected function format_post_list( array $query_results ) {
 				$title .= ' (' . tribe_get_start_date( $wp_post->ID ) . ')';
 			}
 
-			$posts[ $wp_post->ID ] = $title;
+			$posts[ $wp_post->ID ] = apply_filters('the_title', $title, $wp_post->ID);
```
Comment (sc0ttkclark) [inline src/Tribe/Admin/Move_Tickets.php:362]:
```suggestion
			/** This filter is documented in wp-includes/post-template.php */
			$posts[ $wp_post->ID ] = apply_filters( 'the_title', $title, $wp_post->ID );
```

**Comment by sc0ttkclark** [comment]
@jrserani Another PR for G20.09

**Comment by jrserani** [comment]
_Ref:_ [#GTRIA-300](https://moderntribe.atlassian.net/browse/GTRIA-300)

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**